### PR TITLE
refactor: db settings

### PR DESCRIPTION
**Pull request by yesnault**
Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

File: engine/cmd_database.go [hunk 1]
```diff
@@ -212,8 +212,22 @@ func databaseDowngradeCmdFunc(cmd *cobra.Command, args []string) {
...
+	dbConf := database.DBConfiguration{
+		User:           connFactory.DBUser,
+		Role:           connFactory.DBRole,
+		Password:       connFactory.DBPassword,
+		Name:           connFactory.DBName,
+		Schema:         connFactory.DBSchema,
+		Host:           connFactory.DBHost,
+		Port:           int(connFactory.DBPort),
+		SSLMode:        connFactory.DBSSLMode,
+		MaxConn:        connFactory.DBTimeout,
```
Comment (richardlt) [inline engine/cmd_database.go:225]:
```suggestion
		MaxConn:        connFactory.DBMaxConn,
```

File: engine/cmd_database.go [hunk 2]
```diff
@@ -212,8 +212,22 @@ func databaseDowngradeCmdFunc(cmd *cobra.Command, args []string) {
...
+		ConnectTimeout: connFactory.DBConnectTimeout,
+		Timeout:        connFactory.DBMaxConn,
```
Comment (richardlt) [inline engine/cmd_database.go:227]:
```suggestion
		Timeout:        connFactory.DBTimeout,
```

File: engine/cmd_database.go [hunk 3]
```diff
@@ -280,8 +294,20 @@ func databaseStatusCmdFunc(cmd *cobra.Command, args []string) {
...
+	dbConf := database.DBConfiguration{
+		User:           connFactory.DBUser,
+		Role:           connFactory.DBRole,
+		Password:       connFactory.DBPassword,
+		Name:           connFactory.DBName,
+		Schema:         connFactory.DBSchema,
+		Host:           connFactory.DBHost,
+		Port:           int(connFactory.DBPort),
+		SSLMode:        connFactory.DBSSLMode,
+		MaxConn:        connFactory.DBTimeout,
```
Comment (richardlt) [inline engine/cmd_database.go:306]:
```suggestion
		MaxConn:        connFactory.DBMaxConn,
```

File: engine/cmd_database.go [hunk 4]
```diff
@@ -280,8 +294,20 @@ func databaseStatusCmdFunc(cmd *cobra.Command, args []string) {
...
+		ConnectTimeout: connFactory.DBConnectTimeout,
+		Timeout:        connFactory.DBMaxConn,
```
Comment (richardlt) [inline engine/cmd_database.go:308]:
```suggestion
		Timeout:        connFactory.DBTimeout,
```

**Review by richardlt — CHANGES_REQUESTED** [review]

File: engine/database/database.go [hunk 5]
```diff
@@ -105,7 +123,7 @@ func Init(ctx context.Context, user, role, password, name, schema, host string,
 		f.DBName == "" ||
 		f.DBHost == "" ||
 		f.DBPort == 0 {
-		return nil, fmt.Errorf("Missing database infos")
+		return nil, fmt.Errorf("missing database infos")
```
Comment (fsamin) [inline engine/database/database.go:126]:
```suggestion
		return nil, errors.Errorf("missing database infos")
```

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17572.0 ✘  
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✘  
  * Test UI ✔

**Review by fsamin — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

**Pull request merged**